readthedocs/core/management/commands/contact_owners.py

```diff
@@ -0,0 +1,162 @@
+import logging
+import sys
+from pathlib import Path
+from pprint import pprint
+
+import markdown
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+
+from readthedocs.core.permissions import AdminPermission
+from readthedocs.core.utils.contact import contact_users
+from readthedocs.organizations.models import Organization
+from readthedocs.projects.models import Project
+
+User = get_user_model()  # noqa
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+
+    """
+    Send an email or sticky notification from a file (markdown) to all owners.
+
+    Usage examples
+    --------------
```

Comment:
We should probably note the `--production` flag. All of these commands are run in dry-run by default, right?

Reply:
yeah, do you mean to add the `--production` flag in the examples? Or just a note saying that to actually send things, use `--production`?

Reply:
Went for adding the note, so examples aren't copy/pasted/executed by error p: